sentry-raven.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "faraday", ">= 0.7.6"
   gem.add_dependency "certifi"
+  gem.add_dependency "binding_of_caller" if RUBY_VERSION >= '1.9.2' && !defined?(JRUBY_VERSION)
```

Comment:
That sounds good!

Dalli tries to require `kgio` at startup, but rescues the `LoadError` if it fails: https://github.com/mperham/dalli/blob/master/lib/dalli/socket.rb

If we did the same thing here, we could refuse to enable the feature unless `BindingOfCaller` is defined.